solarforecastarbiter/metrics/tests/test_probabilistic.py

```diff
@@ -0,0 +1,143 @@
+import pytest
+import numpy as np
+from solarforecastarbiter.metrics import probabilistic as prob
+
+
+@pytest.mark.parametrize("fx,fx_prob,obs,value", [
+    # forecast 10 MW with 100% probability
```

Comment:
comments should be like "forecast <= 10 MW with 100% probability" correct?

Reply:
Correct. I'll revise to make that clearer (as well as adding notes to the Brier Score docstring).